build/Dockerfile-openjdk8.template

```diff
@@ -57,3 +61,14 @@ WORKDIR ${JANUS_HOME}
 
 ENTRYPOINT [ "docker-entrypoint.sh" ]
 CMD [ "janusgraph" ]
+
+LABEL org.opencontainers.image.title="Janusgraph Docker Image" \
+      org.opencontainers.image.description="Official Janusgraph Docker image" \
+      org.opencontainers.image.url="https://janusgraph.org/" \
+      org.opencontainers.image.documentation="https://docs.janusgraph.org/${JANUS_VERSION}/index.html" \
+      org.opencontainers.image.revision=${REVISION} \
+      org.opencontainers.image.source="https://github.com/JanusGraph/janusgraph-docker/" \
+      org.opencontainers.image.vendor="Janusgraph" \
+      org.opencontainers.image.version=${JANUS_VERSION} \
+      org.opencontainers.image.created=${CREATED} \
+      org.opencontainers.image.license="Apache 2.0"
```

Comment:
This must be a SPDX identifier which is `Apache-2.0` ([Source](https://spdx.org/licenses/Apache-2.0.html)).